src/components/Squeak/components/Replies.tsx

```diff
@@ -108,24 +108,35 @@ type ExpandedProps = {
     replies: StrapiData<ReplyData[]>
 }
 
+const getComunityClasses = (reply, isResolution) => {
+    const profile = reply?.attributes?.profile?.data
+    const isTeamMember = profile?.attributes?.teams?.data?.length > 0
```

Comment:
When a PostHog employee leaves, we remove them from the team. But the answer will still be valid and should hold more weight than others. Should we use the `Start date` field instead as that never changes?